app/src/main/java/com/topjohnwu/magisk/arch/BaseUIActivity.kt

```diff
@@ -54,17 +57,15 @@ abstract class BaseUIActivity<VM : BaseViewModel, Binding : ViewDataBinding> :
             .use { it.getDrawable(0) }
             .also { window.setBackgroundDrawable(it) }
 
-        window?.decorView?.let {
-            it.systemUiVisibility = (it.systemUiVisibility
-                    or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
-                    or View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
-                    or View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION)
-        }
+        window.decorView.windowInsetsController?.hide(WindowInsets.Type.statusBars())
```

Comment:
Using R APIs for all API levels ??? This will crash on any devices lower than R.

If you do want to change this, you should use compact version API from AndroidX, but WindowInsetsController actually behaves different from systemUiVisibility (I haven't deeply investigated it).